app/persistence/message/message.go

```diff
@@ -0,0 +1,37 @@
+package message
+
+import (
+	"gorm.io/gorm"
+)
+
+type TransactionMessage struct {
```

Comment:
You can add the signer address / public key in case you need to check some stuff later.

Reply:
Figured that out earlier today, when coming up with the election process. Thanks